pkgsrc/dev_compile_script.sh

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Tata Consulting
+# Copyright 2016 Ericsson
 #
```

Comment:
a copyright is never changed
if you want to add Ericsson, please append, but not replace not remove.
thanks